helm/polaris/Chart.yaml

```diff
@@ -21,8 +21,8 @@ apiVersion: v2
 name: polaris
 description: A Helm chart for Apache Polaris (incubating)
 type: application
-version: 0.1.0
-appVersion: 1.0.0-incubating-SNAPSHOT
+version: 1.0.0
```

Comment:
Do we want the Helm chart versioning to always follow the versioning of the underlying app?

Reply:
We don't have to, but we need to update it every time we update the helm-chart itself, see details here, https://github.com/apache/polaris/pull/1944#discussion_r2167445639. I'm OK with other version number, any suggestions?

Reply:
Should we move this discussion to the dev ML?

Reply:
Yeah, we did, here are new discussions: https://lists.apache.org/thread/d1vf7xpn6nkzp8gbh417m8qb58tkpcqz